extensions/grapher/src/com/google/inject/grapher/Subgraph.java

```diff
@@ -0,0 +1,26 @@
+package com.google.inject.grapher;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import com.google.inject.Injector;
+import com.google.inject.Key;
+import com.google.inject.spi.ExposedBinding;
+
+/**
+ * Subgraph to represents a private module.
+ *
+ * @author houcheng@gmail.com (Houcheng Lin)
+ */
+public class Subgraph {
```

Comment:
does this need to be a public API or can it be package-private?

also: all the variables should be private, fix the style (don't align the variables vertically, put a blank line btw the vars & the cxtor, no spaces between the type & the type parameters), make the vars final, add getters for the vars.